### PR TITLE
Add app settings import dialog

### PR DESCRIPTION
**Pull request by Flanker32**
Add app settings import dialog

UI:
dialog

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/table/ImportAppSettingsDialog.java [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+    private JButton buttonOK;
+    private JButton buttonCancel;
+    private JComboBox cbAppSettingsSource;
+    private AppSettingsTable tblAppSettings;
+    private JLabel lblAppSettingsSource;
+    private JPanel pnlAppSettings;
+    private JCheckBox chkErase;
+
+    private Path localSettingsPath;
+    private boolean shouldErase;
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/table/ImportAppSettingsDialog.java]: eraseExistingSettings

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/table/ImportAppSettingsDialog.java [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+    @Override
+    public void fillFunctionAppSettings(Map<String, String> appSettings) {
+        tblAppSettings.setAppSettings(appSettings);
+        if (appSettings.size() == 0) {
+            tblAppSettings.getEmptyText().setText(EMPTY_TEXT);
+        }
+    }
+
+    @Override
+    public void prepareFillAppSettings() {
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/table/ImportAppSettingsDialog.java]: beforeLoadAppSettings

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/table/ImportAppSettingsDialog.java [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+
+public class ImportAppSettingsDialog extends JDialog implements ImportAppSettingsView {
+    private static final String LOCAL_SETTINGS_JSON = "local.settings.json";
+    private static final String TITLE_IMPORT_APP_SETTINGS = "Import App Settings";
+    private static final String LOADING_TEXT = "Loading...";
+    private static final String EMPTY_TEXT = "Empty";
+    private JPanel contentPane;
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/table/ImportAppSettingsDialog.java]: shall we make a base dialog class with buttonOK , buttonCancel, contentPanel?

Comment (andxu) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/table/ImportAppSettingsDialog.java]: contentPane -> contentPanel

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/table/ImportAppSettingsDialog.java (hunk 1 above)
Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/table/ImportAppSettingsDialog.java]: Renamed

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/table/ImportAppSettingsDialog.java (hunk 2 above)
Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/table/ImportAppSettingsDialog.java]: Renamed

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/table/ImportAppSettingsDialog.java (hunk 3 above)
Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/table/ImportAppSettingsDialog.java]: Nice idea, will try this later

**Pull request merged**